sdlib/d/gc/slab.d

```diff
@@ -0,0 +1,63 @@
+module d.gc.slab;
+
+import d.gc.emap;
+import d.gc.spec;
+
+enum InvalidBinID = 0xff;
+
+struct SlabAllocGeometry {
+	void* address;
+	size_t size;
```

Comment:
A uint no? Realistically, it all fit on a ushort, but ushort is not gaining anything and typically CPU unfriendly.